webapp-admin/src/components/SigningLog.js

```diff
@@ -57,7 +57,7 @@ class SigningLog extends Component {
       items = this.state.filterModels;
       items[index].selected = !items[index].selected;
       var filter = items.filter(i => i.selected === true).map(i => i.name).join(',')
-      this.getSigningLogs(this.state.authorityID, 0, filter, this.state.query)
```

Comment:
as I updated the method to work without the authorityID I forgot to fix all the calls of the method :(